Update worker dependency

karaxuna opened the pull request:
Change-type: minor

***

Separating worker version bump from this [pr](https://github.com/product-os/jellyfish/pull/4434/files)

**Please remember to write tests for your changes**. We aim to automatically
deploy `master` to production, and we can't safely do this without a solid
test suite!

karaxuna committed:
Update worker dependency

Change-type: minor

karaxuna committed:
Make server test lighter

karaxuna committed:
Typo

karaxuna committed:
Set longer timeout for a test

karaxuna committed:
Set longer timeout for a test

grahammcculloch reviewed (dismissed):
LGTM as long as the tests pass :-)

grahammcculloch committed:
Keep server e2e test actions synchronous

Just an attempt to ease the load on the server while running tests.

Change-type: patch
Signed-off-by: Graham McCulloch <graham@balena.io>
